google/cloud/forseti/scanner/scanners/blacklist_scanner.py

```diff
@@ -113,8 +118,9 @@ def get_instance_networks_interfaces(self):
             network_interfaces.append(ins.create_network_interfaces())
 
         if not network_interfaces:
-            LOGGER.warn('No VM network interfaces found. Exiting.')
-            return None, 0
+            error_message = 'No VM network interfaces found. Exiting.'
```

Comment:
nit: 'Exiting' can be a bit misleading as this isn't doing a fatal log or os.exit here. The exception in a normal situation is handled and ignored in the 'run' function so the binary didn't 'Exit'.